run_scripts/4_coda_add.sh

```diff
@@ -16,13 +16,6 @@ DATA_ROOT=$3
 
 DATASETS=(
     "UNDP_Kenya_s01e01"
-    "UNDP_Kenya_s01e02"
```

Comment:
are this canceled or we want to add them weekly when their data arrives?

Reply:
Will add them later once the process for episode 1 is established and settled.

Reply:
I had removed them everywhere else but here